### ISSUE TITLE
IP BAN

### ISSUE THREAD

**Issue by jhansen0518** [finndev/PokeBuddy#736]
IP Ban wore off so i tried POKEBUDDY again. 1 account, 5km/h.  5 min running fine then IP BAN again.  Will have to wait to see if this can be fixed.

**Comment by Thorben24** [comment]
Strange. Did you changed your ip or just wait some hours?

**Comment by MaartenBloemen** [comment]
You can't use your regular IP in other contries for example if you live in France you have an ip from a french provider, but you start botting in New York, they can see its a wrong IP for America thus they will IP ban you.

**Comment by Krome333** [comment]
how check if we are ban ??

**Comment by Thorben24** [comment]
@MaartenBloemen I dont think so. This IP ban stuff started last night, before there werent any problems like that.

@Krome333 If you login and after you start the bot will no pokestops be displayed / found you are ip banned. And y if you are on your phone actually with the same ip (wifi) than your computer, you can check this this way, too.

**Comment by MaartenBloemen** [comment]
@Thorben24 I couldn't bot in Santa Monica cause i would get an IP ban after 5 minutes but there was no problem when I was botting in Brussels. And now I'm botting in again in Santa Monica using an proxy with an LA IP and no IP ban wathsoever.

**Comment by Thorben24** [comment]
Uhm okay good to know. Im botting actully with 1 german ip (home ip) and 2 proxy servers from france in tokyo, san francisco and sydney. I got no problems yet. But maybe thats a reason for many others who have this problem.

**Comment by AHNMA** [comment]
I can confrim @MaartenBloemen i only bot in Hamburg/Germany and no IP-Ban in hours of farming. Now i tried to play with another account in New-York/USA and i get a IP-Ban within 5min.

I think we got it ;)

**Comment by Thorben24** [comment]
Awesome @AHNMA and @MaartenBloemen . Thats the first two productive posts I am reading today :laughing: Ill try to share this information for guys who have this problem.

**Comment by sqrcbc** [comment]
i live in germany, bot in San Francisco, Disneyland Anaheim, and Germany...
no ip ban lol

2016-08-10 15:53 GMT+02:00 Thorben <notifications@github.com>:

**Comment by Thorben24** [comment]
As I said same for me at the moment, but as we can see many others fixing the problem with that successfully, so its temporary defenetly worth a try.

**Comment by HEROGWP** [comment]
What causes a map no pokestop? I don't understand your mean...

**Comment by Thorben24** [comment]
@HEROGWP 
This mean your ip adress is banned. Wait for 1-2 hours! After that try to bot in the country / state you live.

**Comment by sqrcbc** [comment]
moving / Teleporting too fast causes you getting soft banned
soft ban => no pokestops farmable, no pokemons catchable

or do you mean you dont see any pokestop?
shall that be an IP ban?

2016-08-10 16:05 GMT+02:00 蔡建弘 <notifications@github.com>:

**Comment by Thorben24** [comment]
If he dont see pokestops -> IP ban, if he get 0 xp by pokestops -> Soft ban

**Comment by matt2310** [comment]
@Thorben24  i thought the IP bans were only a soft ban also? or are these permanent?

**Comment by HEROGWP** [comment]
yes, I'm not sure whether my IP be banned.
If i don't want wait for 1-2 hours, I have to change my IP(use proxy)?

**Comment by nguyeneer** [comment]
So how do we change our IP?

**Comment by Thorben24** [comment]
IP bans are usually just something around 1 hour. Soft bans usually longer, more than 6 hours.

Generally info for IP bans, so in case you can not find pokestops anymore:
If there are no pokestops on the map it is an IP-Ban. Wait 1-6 hours. If you don't want to wait You have to change your IP adress in this case to solve this problem. 95% of the providers use dynamic ip adressing, so a relog / restart of your router should renew your ip adress. Usually the IP ban will only happen if you bot / play more than 1 account at the same time. In the future you should only use 1 ip adress for 1 account. For others you would have to use proxy tunnels each pokebuddy instance you are running. Right now, if you can not switch your ip adress, you can login via a VPN. Here is a great tutorial for that: You can also just use a vpn for logging in and switching back to your home adress after it. Here is a great tutorial: http://www.ownedcore.com/forums/pokemon-go/pokemon-go-hacks-cheats/570331-quick-ip-unban-vpn-ip-no-ban.html

**Comment by matt2310** [comment]
@Thorben24  so if i wait, i just try again on my home IP and just hope i dont get banned again?

**Comment by nguyeneer** [comment]
Ah...
I logged in at my Uni this morning, which is thirty minutes from my home, but I've been using it from home before now.
So, because of the different IP, I got IP banned?

**Comment by Thorben24** [comment]
Usually this shouldnt be the problem. But it can be. Right now i can just assume to wait for an pokebuddy update.

**Comment by nguyeneer** [comment]
How do you check if there's an update?
Sorry for all theses questions...
Kinda new to GitHub

**Comment by Thorben24** [comment]
Click: https://github.com/finndev/PokeBuddy
There you can see. Latest commit ea5478b  16 hours ago @EloBuddyLtd EloBuddyLtd Update PokeBuddy

If this text / time is chaning, you can just restart pokebuddy and it automatically update.

**Comment by nguyeneer** [comment]
Thanks so much!
:)
